source/includes/_invoice.md

```diff
@@ -1449,11 +888,7 @@ invoiceApi.get_invoice_as_html(invoice_id, api_key, api_secret)
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 200 OK
-< Content-Type: text/html
-
+```
```

Comment:
should this be ```json?

Reply:
Hi Reshma, this is retrieving invoice as html. 

![image](https://github.com/killbill/slate/assets/75751774/06a6551b-05db-4c1d-87b4-d654d454582e)